### PR TITLE
Fix TileSet painting options appear out of screen

### LINKED ISSUE #83610: TileSet painting options appear out of screen
### Godot version

4.2 beta1

### System information

Windows 10.0.19045 - Vulkan (Forward+) - dedicated NVIDIA GeForce GTX 1060 (NVIDIA; 30.0.15.1403) - Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz (8 Threads)

### Issue description

https://github.com/godotengine/godot/assets/2223172/c8442a7e-7223-442d-8803-5fdf884b5f6e

### Steps to reproduce

1. Create TileMap + TileSet
2. Go to Paint tab
3. Click Select a property editor
4. The options appear partially obstructed
5. Click again
6. Options are fully visible now

### Minimal reproduction project

N/A

### PR DESCRIPTION
**Pull request by OppositeNor**
This is a fix for issue TileSet painting options appear out of screen issue [#83610](https://github.com/godotengine/godot/issues/83610)

*Bugsquad edit:*
- Fixes #83610

### CODE REVIEW

File: editor/plugins/tiles/tile_set_atlas_source_editor.cpp [hunk 1]
```diff
@@ -836,6 +836,9 @@ void TileSetAtlasSourceEditor::_update_tile_data_editors() {
 		item->set_custom_color(0, disabled_color);
 	}
 
+
```
Comment (AThousandShips) [inline editor/plugins/tiles/tile_set_atlas_source_editor.cpp:839]:
```suggestion
```
Too many lines

**Comment by KoBeWi** [comment]
A better solution would be changing this button to a proper OptionButton, but it's not that easy, so this solution works too for now.
Although seems like you did a wrong push and deleted the PR .-.

**Comment by OppositeNor** [comment]
Um... Yeah I think I accidently deleted it.

**Comment by OppositeNor** [comment]
I'll open a new one and try again.

**Comment by AThousandShips** [comment]
Superseded by:
* https://github.com/godotengine/godot/pull/83790

**Pull request closed without merging**